### DISCUSSION TITLE
Can't use a Bunafr Roast Profile with Hottop Not Communicating with Artisan and Hottop Roaster

### DISCUSSION THREAD

**Discussion by thedirtyduo** [artisan-roaster-scope/artisan#1657] [Q&A]
I'm on a brand new PC laptop with a Hottop 2k+. I'm also on the latest update 3.0.0 now because I was having the same issue with 2.10. After updating I rebooted my computer.

I setup Config>Machine and selected the right roaster and port COMM3. I have no other cables plugged into my laptop. I also Load the Bunafr roast profile. I activated Super ON. The next step on my old Mac laptop was then to hit Control then Start. Clicking on Control does nothing. I know it's not the usb cable because I can control the roaster but then I'm in full manual mode and it's not using the Bunafr Roasting profile. 

I must be missing a crucial step.

**Comment by poundy** [comment]
Did you review older posts here? 
https://github.com/artisan-roaster-scope/artisan/discussions/1441

2.10 bought some changes to the architecture.
